tests/unit/test_utils.py

```diff
@@ -416,16 +426,43 @@ def test_convert_units_to_MtCO2_equiv_works(check_aggregate_df, ARoption, expect
     )
 
 
+def test_convert_units_to_MtCO2_equiv_equiv_start(check_aggregate_df):
+    # Check that it does nothing when nothing needs doing
+    limited_check_agg = check_aggregate_df.filter(
+        variable="Primary Energy*", keep=False
+    )
+    limited_check_agg.data["unit"] = "kt CF4-equiv/yr"
+    converted_data = convert_units_to_MtCO2_equiv(limited_check_agg)
+
+    assert (converted_data.data["unit"] == "Mt CO2-equiv/yr").all()
+
+    with _ur.context("AR5GWP100"):
+        exp_conv_factor = _ur("kt CF4/yr").to("Mt CO2/yr").magnitude
+    assert converted_data.data["value"].equals(limited_check_agg.data["value"] * exp_conv_factor)
+
+
 def test_convert_units_to_MtCO2_equiv_doesnt_change(check_aggregate_df):
     # Check that it does nothing when nothing needs doing
     limited_check_agg = check_aggregate_df.filter(
         variable="Primary Energy*", keep=False
     )
-    limited_check_agg.data["unit"] = "Mt CO2 equiv/yr"
+    limited_check_agg.data["unit"] = "Mt CO2-equiv/yr"
     converted_data = convert_units_to_MtCO2_equiv(limited_check_agg)
     assert converted_data.data.equals(limited_check_agg.data)
 
 
+def test_convert_units_to_MtCO2_becomes_MtCO2_equiv(check_aggregate_df):
+    # Check that it does nothing when nothing needs doing
```

Comment:
You are actually checking that you do something when nothing needs doing here

Reply:
Will update depending on your thoughts on the above

Reply:
updated in 9aaf021d9

Reply:
Some things never chnage.